src/library_browser.js

```diff
@@ -1421,13 +1421,19 @@ var LibraryBrowser = {
 
   // Runs natively in pthread, no __proxy needed.
   emscripten_async_call__sig: 'viii',
+  emscripten_async_call__deps: ['$safeSetTimeout'],
   emscripten_async_call: function(func, arg, millis) {
     function wrapper() {
       {{{ makeDynCall('vi', 'func') }}}(arg);
     }
 
+#if ENVIRONMENT_MAY_BE_NODE
+    // node does not support requestAnimationFrame
+    if (millis >= 0 || ENVIRONMENT_IS_NODE) {
```

Comment:
The `millis >= 0` check should not be in the ifdef I think.

Reply:
I think I needs to be since `ENVIRONMENT_MAY_BE_NODE` is true by default isn't it?   If we want to use safeRequestAnimationFrame in the browser when `ENVIRONMENT_MAY_BE_NODE` is set don't we need it?

Reply:
Oh, sorry, I see the millis part is duplicated, which confused me. I was looking for this:
```js
  if (millis >= 0
#ifdef NODE
      || NODE
#endif
    ) {
```

I lean towards that to reduce duplication, but I don't feel strongly.

Reply:
OK will do.   I'm a little allergic to using `#ifdef` to break up  single line like that, but maybe you are right.